src/test/main_tests.cpp

```diff
@@ -73,4 +74,127 @@ BOOST_AUTO_TEST_CASE(test_combiner_all)
     BOOST_CHECK(Test());
 }
 
+// Older version of the merkle root computation code, for comparison.
```

Comment:
I don't know, checkblock_tests.cpp? maybe its own merkle_test.cpp ?
The only use in main.cpp is in CheckBlock(), which should eventually move to the consensus dir...